Game quits when you collect 4 orbs. (WIP)

DSF256 opened the pull request:
The game should quit whenever you collect 4 orbs (i.e., finish the level).

This is a WIP, obviously.  The game should move onto level 2, but I haven't completed that requirement yet.  Funny how we're still doing something related to waterfall even though we're using agile. :/

Btw, this took WAY too long for me to figure out how to do without breaking the rest of the game. :/

DSF256 committed:
Game quits when you collect 4 orbs. (WIP)

Btw, this took WAY too long for me to figure out how to do without breaking the rest of the game. :/

reid-wynkoop commented on 2d/platformer/src/Actors/Player.gd:
```diff
@@ -155,6 +155,8 @@ func _on_coinCollected():
 func _on_orbCollected():
 	self.orbs = self.orbs + 1
 	$UI/amtOrbsLabel.text = "Orbs: " + String(self.orbs)
+	if (self.orbs >= 4):
```
Should have a constant global var for the amount of orbs needed.

mbagwell replied:
It says the code is outdated for Player.gd, might want to resolve that first.

reid-wynkoop replied:
@mbagwell that just means he made a commit to it after I made a comment. In his commit, he created a constant var like I suggested, so it is fixed.

reid-wynkoop commented on 2d/platformer/src/Actors/Player.gd:
```diff
@@ -169,7 +171,12 @@ func _collideWithEnemyCheck():
+func _orbCheck():
+	# for now, the game will just quit whenever you collect 4 orbs
+	# obviously, that'll get changed later
+	get_tree().quit()
```
Agree, for now this is fine.

DSF256 committed:
Muted pickup audio (it causes bugs); replaced the orb sprite with something better.

Same as above; also made the orb condition a constant (ORBS_NEEDED) instead of a variable (4.0)

mbagwell reviewed (approved):
The orb function makes sense, when the user gets x amount of orbs the  game quits.
